modules/react/common/lib/styles/focusRing.ts

```diff
@@ -62,7 +61,8 @@ function calculateFocusRing({
   return {boxShadow};
 }
 
-export const memoizedFocusRing = memoize(calculateFocusRing, (...args) => JSON.stringify(args));
+// Is memoize really needed here??
+export const memoizedFocusRing = calculateFocusRing;
```

Comment:
If we need memorize we could just inline the lodash function `https://github.com/lodash/lodash/blob/master/memoize.js` its pretty short

Reply:
I don't think this needs to be memoized. I haven't found anything that warrants a stable reference.